### PR TITLE
[No card] Upgrade slate and slate-react

### PR DESCRIPTION
**Pull request by dangtranEH**
- We face a lot of issues related to slate in Sentry. However, we cannot reproduce it in the local/sandbox to detect and fix it. After researching, updating the slate version is the only way we can think of.

Sentry issues:
- https://employment-hero.sentry.io/issues/4377654947/?alert_rule_id=11445288&alert_type=issue&project=210777&referrer=slack
- https://employment-hero.sentry.io/issues/4377654947/?referrer=slack&alert_rule_id=11445288&alert_type=issue
- https://employment-hero.sentry.io/issues/4358282959/?alert_rule_id=11445288&alert_type=issue&project=210777&referrer=slack
- https://employment-hero.sentry.io/issues/3059672356/?alert_rule_id=11445288&alert_type=issue&project=210777&referrer=slack

Repo discussion:
- https://github.com/ucfopen/Obojobo/issues/1469

Screen recording to make sure the editor works after upgrading:

https://github.com/Thinkei/hero-editor/assets/124115080/1a474f26-756a-4593-8022-5567083509f8

https://github.com/Thinkei/hero-editor/assets/124115080/190dc368-6ca3-4af5-a7a7-a683526cf8a4

https://github.com/Thinkei/hero-editor/assets/124115080/6444bd84-354b-47fd-9926-1e46f20bf3f0

### CODE REVIEW

**Comment by dangtranEH** [comment]
Hi a @ttkien. Should we consider upgrading these packages to eliminate Sentry errors?

**Comment by ttkien** [comment]
Of course. Just remind that slate js is used by web and mobile (RichTextEditor). So, we should do testing carefully @dangtranEH

**Comment by dangtranEH** [comment]
> Of course. Just remind that slate js is used by web and mobile (RichTextEditor). So, we should do testing carefully @dangtranEH

What should we do next, a @ttkien? Merge, publish a new version, upgrade, and test for web and mobile, or do anything else?

**Comment by ttkien** [comment]
> > Of course. Just remind that slate js is used by web and mobile (RichTextEditor). So, we should do testing carefully @dangtranEH
> 
> 
> 
> What should we do next, a @ttkien? Merge, publish a new version, upgrade, and test for web and mobile, or do anything else?

For mobile, test on android and ios both.

File: lib/src/lib.js [hunk 1]
```diff
@@ -137,6 +139,7 @@ const HeroEditor = ({
       <Slate
         editor={editor}
         value={value}
+        initialValue={initialValue}
```
Comment (dangtranEH) [inline lib/src/lib.js:142]: initialValue is required in the new version.

**Comment by dangtranEH** [comment]
@ttkien I have tested the editor on IOS/Android. Please help me review it again. Thank you.

**Review by ttkien — APPROVED** [review]

**Review by briankhdo — APPROVED** [review]

**Pull request merged**